sktime/libs/granite_ttm/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
redundant with the `__init__`?

Reply:
yes, its redundant, I've removed the LICENSE file